gui/default/index.html

```diff
@@ -752,10 +753,8 @@ <h4 class="panel-title">
                   <table class="table table-condensed table-striped table-auto">
                     <tbody>
                       <tr class="visible-xs">
-                        <th>
-                          <span class="fa fa-fw {{deviceStatusIcon(deviceCfg)}}"></span>&nbsp;<span translate>Device Status</span>
-                        </th>
-                        <td class="text-right">{{deviceStatusText(deviceCfg)}}</td>
+                        <th><span class="fa fa-fw {{deviceStatusIcon(deviceCfg)}}"></span>&nbsp;<span translate>Device Status</span>
+                        </th><td class="text-right">{{deviceStatusText(deviceCfg)}}</td>
```

Comment:
This doesn't look right... The `</th>` definitely doesn't belong on this line. :wink:

```suggestion
                        <th><span class="fa fa-fw {{deviceStatusIcon(deviceCfg)}}"></span>&nbsp;<span translate>Device Status</span></th>
                        <td class="text-right">{{deviceStatusText(deviceCfg)}}</td>
```